server/remote/gitea/gitea.go

```diff
@@ -458,7 +461,13 @@ func (c *Gitea) newClientToken(ctx context.Context, token string) (*gitea.Client
 			TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
 		}
 	}
-	return gitea.NewClient(c.URL, gitea.SetToken(token), gitea.SetHTTPClient(httpClient), gitea.SetContext(ctx))
+	client, err := gitea.NewClient(c.URL, gitea.SetToken(token), gitea.SetHTTPClient(httpClient), gitea.SetContext(ctx))
```

Comment:
So it will try to auto detect the version and use `v1.17.0` if it fails?

Reply:
right

Reply:
I wont like to make it a config option, as you only should have this problem if you develop gitea ...